tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36, py37, py38, py39, py310, py311, pypy3
+envlist = py310, py311, py312, pypy3
```

Oh good point! Thanks for updating this 👍 
I indeed see `py36` could have been removed already a while back 😅 

Maybe that's too picky from my side, but I feel a bit more comfortable keeping `py37, ... py39` for now, do you mind still including these? 

```suggestion
envlist = py37, py38, py39, py310, py311, py312, pypy3
```

According to [the `pypistats`](https://pypistats.org/packages/pypiserver) we are still getting some (albeit not much) downloads from these Python versions. I'd like to remove the `tox` configuration for them separately.